webapp/src/components/AcceptBidAssetPage/AcceptBidAssetPage.container.js

```diff
@@ -0,0 +1,72 @@
+import { withRouter } from 'react-router'
+import { connect } from 'react-redux'
+import { push } from 'react-router-redux'
+import { getAddress, isConnected } from '@dapps/modules/wallet/selectors'
+
+import { locations } from 'locations'
+import { ASSET_TYPES } from 'shared/asset'
+import { buildCoordinate } from 'shared/coordinates'
+import {
+  getMatchParams,
+  getMatchParamsCoordinates
+} from 'modules/location/selectors'
+import { acceptBidRequest, fetchBidByIdRequest } from 'modules/bid/actions'
+import {
+  getBidById,
+  isCancelIdle,
+  isLoading as isBidLoading
+} from 'modules/bid/selectors'
+import AcceptBidAssetPage from './AcceptBidAssetPage'
+
+const mapState = (state, ownProps) => {
+  let assetId
+  const { bidId, id } = getMatchParams(ownProps)
+  assetId = id
```

Comment:
You can just let here
```suggestion
  let assetId = id
```